exporters/pelorus/__init__.py

```diff
@@ -33,7 +33,7 @@ def _print_version():
     exporter_name = pathlib.PurePath(__main__.__file__).parent.name
 
     repo, ref, commit = (
-        os.environ.get(f"OPENSHIFT_BUILD_{var.upper()}")
+        utils.get_env_var(f"OPENSHIFT_BUILD_{var.upper()}")
```

Comment:
honest question... what is the advantage of utils.get_env_var over os.environ.get?

Reply:
It's explained in the utils.get_env_var function, there are two reasons:

1. Have consistent fallback option when we will start using config maps, so we don't have case where SOME_VAR="" is considered non empty value and instead of default one empty string is used.
2. Have one function where we will change the behavior if needed across all usages in pelorus.